django/contrib/postgres/lookups.py

```diff
@@ -60,3 +60,24 @@ def process_lhs(self, qn, connection):
             self.lhs = SearchVector(self.lhs)
         lhs, lhs_params = super(SearchLookup, self).process_lhs(qn, connection)
         return lhs, lhs_params
+
+
+class TrigramSimilar(PostgresSimpleLookup):
+    lookup_name = 'trigram_similar'
+    operator = '%%'
+
+
+class TrigramDistance(PostgresSimpleLookup):
```

Comment:
We definitely want to have a way of saying `.filter(TrigramDistance(F('foo'), 'test') > 0.7)` or something, but we haven't been able to come up with the right syntax yet so annotate is the compromise.

I wasn't aware of the geo lookups, they were written a long time before expressions existed when that would have been the only way of implementing them. GeoDjango in general is in need of a lot of tidy up in a post-expressions world.